pipeline_dp/budget_accounting.py

```diff
@@ -100,7 +100,182 @@ def compute_budgets(self):
         for requested_budget in self._requested_budgets:
             eps = delta = 0
             if total_weight_eps:
-                eps = requested_budget.use_eps * self._eps * requested_budget.weight / total_weight_eps
+                numerator = requested_budget.use_eps * self._eps * requested_budget.weight
+                eps = numerator / total_weight_eps
             if total_weight_delta:
-                delta = requested_budget.use_delta * self._delta * requested_budget.weight / total_weight_delta
+                numerator = requested_budget.use_delta * self._delta * requested_budget.weight
```

Comment:
👋  Changing a couple of things in this file due to failing lint (lines too long)